src/main/java/org/sagebionetworks/bridge/spring/controllers/FileController.java

```diff
@@ -88,4 +93,37 @@ public StatusMessage deleteFile(@PathVariable String guid,
         }
         return DELETE_MSG;
     }
+    
+    @GetMapping("/v3/files/{guid}/revisions")
+    public PagedResourceList<FileRevision> getFileRevisions(@PathVariable String guid,
+            @RequestParam(required = false) String offsetBy, @RequestParam(required = false) String pageSize) {
+        UserSession session = getAuthenticatedSession(DEVELOPER);
+        
+        int offsetInt = BridgeUtils.getIntOrDefault(offsetBy, 0);
+        int pageSizeInt = BridgeUtils.getIntOrDefault(pageSize, API_DEFAULT_PAGE_SIZE);
+
+        return fileService.getFileRevisions(session.getStudyIdentifier(), guid, offsetInt, pageSizeInt);
+    }
```

Comment:
Where are the unit tests for getFileRevisions()?

Reply:
totally missed these... added.